genie_registry/clinical.py

```diff
@@ -493,18 +487,16 @@ def _validate(self, clinicaldf, oncotree_link):
         clinicaldf = clinicaldf.fillna("")
 
         oncotree_mapping_dict = process_functions.get_oncotree_code_mappings(
-            oncotree_link
+            self.genie_config["oncotreeLink"]
```

Comment:
Is `genie_config` being passed to file objects upon instantiation?  When running the command 
`genie validate --project_id syn7208886 data_clinical_supp_SAGE.txt SAGE`

I get the following error
 

> Traceback (most recent call last):
>   File "/Users/hhunterzinck/opt/anaconda3/envs/genie/bin/genie", line 8, in <module>
>     sys.exit(main())
>   File "/Users/hhunterzinck/opt/anaconda3/envs/Genie/lib/python3.8/site-packages/genie/__main__.py", line 176, in main
>     args.func(syn, args)
>   File "/Users/hhunterzinck/opt/anaconda3/envs/Genie/lib/python3.8/site-packages/genie/validate.py", line 263, in _perform_validate
>     valid, message = validator.validate_single_file(**mykwargs)
>   File "/Users/hhunterzinck/opt/anaconda3/envs/Genie/lib/python3.8/site-packages/genie/validate.py", line 105, in validate_single_file
>     valid, errors, warnings = validator.validate(
>   File "/Users/hhunterzinck/opt/anaconda3/envs/Genie/lib/python3.8/site-packages/genie/example_filetype_format.py", line 178, in validate
>     errors, warnings = self._validate(df, **mykwargs)
>   File "/Users/hhunterzinck/opt/anaconda3/envs/Genie/lib/python3.8/site-packages/genie_registry/clinical.py", line 490, in _validate
>     self.genie_config["oncotreeLink"]
> TypeError: 'NoneType' object is not subscriptable
>

Reply:
This is an error on my part. Need to add to the validate cli.